locales/locales/en.json

```diff
@@ -564,6 +564,12 @@
 	"keyboardNavMoveToParentDesc": {
 		"message": "Move to parent (in comments)."
 	},
+	"keyboardNavUndoMoveTitle": {
+		"message": "Undo Move"
+	},
+	"keyboardNavUndoMoveDesc": {
+		"message": "Move to the previously selected thing."
```

Comment:
Maybe note that it only works once, i.e. you can't ctrl-z all the way back to the OP. Not sure the best way to phrase that, though.